chromadb/test/property/test_persist.py

```diff
@@ -70,7 +70,10 @@ def test_persist(
     collection_strategy: strategies.Collection,
     embeddings_strategy: strategies.RecordSet,
 ) -> None:
-    api_1 = chromadb.Client(settings)
+    system_1 = System(settings)
```

Comment:
i understand what the code in this file is doing, but why was it necessary to make this change?

Reply:
System allows us to stop() which frees sqlite connections.